app.css

```diff
@@ -103,3 +103,350 @@ p {
   align-items: center;
   margin: 0% 0% 0% 7%;
 }
+
+/* container for all works */
+
+.works {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: 250px repeat(3, auto);
+  justify-content: center;
+  width: 100%;
+  height: 430vh;
+  padding-left: 24px;
+  padding-right: 24px;
+  background: #fff;
+}
+
+/* container for Recent works title */
+
+.title {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  padding-top: 80px;
+  padding-bottom: 80px;
+  align-items: center;
+  margin-top: 10%;
+}
+
+/* Heading for Recent Works */
+
+.recentworks {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  text-align: center;
+  padding-bottom: 20px;
+  width: 327px;
+  height: 83px;
+}
+
+.works1 {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: -5% 0% 0% 0%;
+}
+
+.work1Title {
+  width: 327px;
+  height: 36px;
+  color: #091e42;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  padding-top: 24px;
+}
+
+.work1Description {
+  width: 327px;
+  height: 147px;
+  color: #091e42;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 24px;
+  text-align: left;
+  margin: -2% 5% 0% -1%;
+}
+
+.work1Languages {
+  width: 314px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  top: 207px;
+  gap: 12px;
+  list-style: none;
+  margin: 10% 0% 0% -1%;
+}
+
+.work1Button {
+  background-color: #fff;
+  flex-direction: row;
+  padding: 10px 12px;
+  align-items: center;
+  box-sizing: border-box;
+  height: 40px;
+  border: 2px solid #8993a4;
+  text-decoration: none;
+  display: inline-block;
+  font-weight: 400;
+  font-size: 15px;
+  margin: 10% 0% 0% 2%;
+}
+
+.Works1Link {
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 15px;
+  cursor: pointer;
+  width: 124px;
+  margin: 6% 1% 1% -1%;
+}
```

Comment:
- [ ] Please make sure the `see project` button looks like the button in [figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=ULC4CUKEW0pdVT14-0)

Figma design:

<img width="216" alt="Screenshot 2023-04-01 at 9 32 48 AM" src="https://user-images.githubusercontent.com/44798044/229267107-d000a1bf-667b-4048-bc88-53c9f6035605.png">

Your design:

<img width="157" alt="Screenshot 2023-04-01 at 9 31 45 AM" src="https://user-images.githubusercontent.com/44798044/229267113-d9612182-2b37-4187-bd6d-9e7f914bdb39.png">